src/main/java/reposense/parser/CsvParser.java

```diff
@@ -51,105 +51,107 @@ public CsvParser(Path csvFilePath) throws IOException {
      */
     public List<T> parse() throws IOException {
         List<T> results = new ArrayList<>();
+        Iterable<CSVRecord> records;
 
-        try (BufferedReader br = new BufferedReader(new FileReader(csvFilePath.toFile()))) {
-            // Skip first line, which is the header row
-            br.readLine();
-            String line;
-
-            for (int lineNumber = 2; (line = br.readLine()) != null; lineNumber++) {
-                String[] elements = line.split(ELEMENT_SEPARATOR);
+        try {
+            Reader csvReader = new FileReader(csvFilePath.toFile());
```

Comment:
`FileReader` implements `AutoCloseable`.

Should use try-with-resources:
 `try (Reader csvReader = new FileReader(csvFilePath.toFile()))`